src/main/resources/application.yml

```diff
@@ -83,4 +83,9 @@ logging:
 # At the moment the treatment codes aren't finalised so there's an example one
 # that tests it. Once the direct delivery treatment codes have been finalised then we can add
 # them to the list.
-directdeliverytreatmentcodes: CE_LDIEE
\ No newline at end of file
+directdeliverytreatmentcodes: CE_LDIEE
+
+# Skeleton cases need to know the collection exercise id and action_plan_id
+censusconfig:
```

Comment:
"config" is not very descriptive. Obviously this is config because it's in a config file. Could you maybe call it "censusskeleton". Not super bothered but "censusconfig" is so generic it's a bit meaningless.